includes/class-wcpdf-admin.php

```diff
@@ -1360,6 +1364,47 @@ public function add_invoice_number_value_to_order_export( $export_item, $item )
 
 		return $export_item;
 	}
+
+	public function add_invoice_column_to_sortable_columns( array $columns ): array {
+		$columns['invoice_date_column']   = 'invoice_date_column';
+		$columns['invoice_number_column'] = 'invoice_number_column';
+
+		return $columns;
+	}
+
+	public function adjust_order_list_query_args( array $order_query_args ): array {
+		if ( 'invoice_number_column' === $order_query_args['orderby'] ) {
+			$order_query_args['meta_query'] = array(
+				'invoice_number_column' => array(
+					'key'     => '_wcpdf_invoice_number',
+					'compare' => '!=',
+					'value'   => '0',
+					'type'    => 'NUMERIC',
+				),
+			);
+		}
+
+		if ( 'invoice_date_column' === $order_query_args['orderby'] ) {
+			$order_query_args['meta_query'] = array(
+				'invoice_date_column' => array(
+					'key'     => '_wcpdf_invoice_date',
+					'compare' => '!=',
+					'value'   => '',
+				),
+			);
+		}
+
+		return $order_query_args;
+	}
+
+	public function sort_orders_by_numeric_invoice_number( $query ): void {
+		if ( ! is_admin() || ! $query->is_main_query() || 'shop_order' !== $query->get( 'post_type' ) && '_wcpdf_invoice_number' !== $query->get( 'meta_key' ) ) {
+			return;
+		}
+
+		$query->set( 'orderby', 'meta_value_num' );
```

Comment:
I think we should wrap `meta_value_num` within the `wpo_wcpdf_invoice_number_column_orderby` filter hook, to let the customer switch to `meta_value`, if they are using a prefix/suffix, e.g. `INV-000123/2024`.